tests/integration.rs

```diff
@@ -72,27 +72,6 @@ const DIRECT_KERNEL_BOOT_CMDLINE: &str =
 
 const CONSOLE_TEST_STRING: &str = "Started OpenBSD Secure Shell server";
 
-fn prepare_virtiofsd(
-    tmp_dir: &TempDir,
-    shared_dir: &str,
-    cache: &str,
-) -> (std::process::Child, String) {
-    let virtiofsd_socket_path =
-        String::from(tmp_dir.as_path().join("virtiofs.sock").to_str().unwrap());
-
-    // Start the daemon
-    let child = Command::new("virtiofsd")
-        .args(&[format!("--socket-path={}", virtiofsd_socket_path).as_str()])
-        .args(&["-o", format!("source={}", shared_dir).as_str()])
-        .args(&["-o", format!("cache={}", cache).as_str()])
-        .spawn()
-        .unwrap();
-
-    thread::sleep(std::time::Duration::new(10, 0));
-
-    (child, virtiofsd_socket_path)
-}
-
 fn prepare_virtiofsd_rs_daemon(
```

Comment:
Since we now consider `virtiofsd` as the Rust version by default, I think we should rename `prepare_virtiofsd_rs_daemon` into `prepare_virtiofsd`.